order/buyer/receive-goods.md

```diff
@@ -32,15 +32,29 @@ Before adding a delivery to an order, the order must have been sent to Tradeclou
 {% endhint %}
 
 {% hint style="warning" %}
-When using the single delivery per order line feature, the deliveries endpoint only supports the first order line of each item number. The deliveries of the first line will be applied to all order lines having the same item, prices and terms.
+**Single delivery order line behavior for delivery events:**
```

Comment:
Same comment as for the indicators API. 
Also add a hint if someone want a Delivery history in the single end point https://tradecloud.atlassian.net/browse/TC-9954

Reply:
updated